lib/video/postprocessing/visualmetrics/visualMetrics.js

```diff
@@ -16,9 +16,29 @@ const SCRIPT_PATH = path.join(
   'visualmetrics.py'
 );
 
+const PORTABLE_SCRIPT_PATH = path.join(
+  __dirname,
+  '..',
+  '..',
+  '..',
+  '..',
+  'browsertime',
+  'visualmetrics-portable.py'
+);
+
+async function getScript(options) {
+  if (options.visualMetricsPortable) {
+    return PORTABLE_SCRIPT_PATH;
+  }
+  return SCRIPT_PATH;
+}
+
 module.exports = {
-  async checkDependencies() {
-    return execa(process.env.PYTHON || 'python', [SCRIPT_PATH, '--check']);
+  async checkDependencies(options) {
```

Comment:
@soulgalore is this method used anywhere? I wasn't able to find any references to it.